app/src/main/java/sdp/moneyrun/LeaderboardActivity.java

```diff
@@ -43,7 +53,7 @@ public void addPlayerList(ArrayList<Player> playerList){
     public void addPlayer(Player player){
```

Comment:
You should consider adding documentation

Reply:
Will do